app/scripts/modules/core/search/search.service.js

```diff
@@ -1,42 +0,0 @@
-'use strict';
-
-let angular = require('angular');
-
-import {SETTINGS} from 'core/config/settings';
-
-module.exports = angular.module('spinnaker.core.search.service', [])
-  .factory('searchService', function($q, $http, $log) {
-
-    var defaultPageSize = 500;
-
-    function getFallbackResults() {
-      return { results: [] };
-    }
-
-    function search(params) {
-      var defaultParams = {
-        pageSize: defaultPageSize
-      };
-
-      return $http({
-        url: SETTINGS.gateUrl + '/search',
-        params: angular.extend(defaultParams, params),
-        timeout: SETTINGS.pollSchedule * 2 + 5000, // TODO: replace with apiHost call
```

Comment:
The API service uses the same timeout, so I just used `API` directly instead of `$http` here.